aurget

```diff
@@ -554,7 +565,7 @@ setup_targets() {
     version="${versions[$name]}"
 
     if [[ -z "$version" ]]; then
-      pkgbuild="$temp_directory/${name}_PKGBUILD"
+      pkgbuild="$pkgbuild_directory/${name}_PKGBUILD"
```

Comment:
I would place the file at `$cache/$name/$version/PKGBUILD` -- You're going to need `$version` in the filename to find it later for `diff`ing. But... you don't actually know the `version` yet, so you might still have to download it to some temporary location and move it after processing. I'm open to ideas here.

Reply:
That was my thought as well, maybe there is a way to hold it in a folder and parse it and then moving it into it's own directory in name/version before continuing?